src/util/math-util.js

```diff
@@ -41,6 +41,9 @@ class MathUtil {
      */
     static wrapClamp (n, min, max) {
         const range = (max - min) + 1;
+        if (!isFinite(n)) {
+            return min;
+        }
```

Comment:
I'm sorry @TheBrokenRail, I didn't really think through what I was asking you to "just move to `wrapClamp`". I think the behavior you had before actually was more appropriate (leave the direction unchanged).  Would you please change it back to what you had? I think that makes the most sense for now.  Thanks for the quick update!